### DISCUSSION TITLE
All Users can create Asset-Folders at the root

### DISCUSSION THREAD

**Discussion by P4mes4n** [requarks/wiki#6585] [Error / Bug Report]
OS: Unraid 6.11.5 and Docker
Wiki.js version: 2.5.299
Database engine: PostgreSQL 15

It seems like the permissions aren't working as intended in this regard.
If you set up a User group `Test` with all content permissions with `path starts with...: /test` they should only be able to interact with anything starting only at that path.

Currently if you open the "insert asset" dialogue, you always start at the root. The members can't see any other folder (or assets), except `test`. They can however select `create new folder`. When they do that, it shows as successful. They can't see that folder, but it still gets created. If you log in via Admin you can see the created folder and can use it to upload assets.

This is particularly annoying, because you can't easily remove folders. AFAIK it requires messing with the DB.
Am I missing something?

**Comment by NGPixel** [comment]
This is a limitation in v2 and will be addressed in v3.

**Comment by tofixx** [comment]
The permission checks are not fully implemented for all folder actions. Why?!

**Comment by MaBauMeBad** [comment]
Are there any updates on this ?

Would it also a solution for you @NGPixel , to bind the create Folder action to the "Upload asset" permission for the time being ?
Afaik, currently it is more then an inconvenience, since we have no valid solution to maintain the asset folder without to modify the database (which is a bad idea).

Thank you.
